swan-cern/scripts/before-notebook.d/10_spark.sh

```diff
@@ -37,7 +37,7 @@ then
     source $SPARK_CONFIG_SCRIPT $SPARK_CLUSTER_NAME $HADOOPVERSION $SPARKVERSION
   fi
 
-  if [[ $CONNECTOR_BUNDLED_CONFIGS ]]
+  if [[ $CONNECTOR_BUNDLED_CONFIGS && -d "$JUPYTER_CONFIG_DIR/nbconfig" ]]
   then
     ln -s $CONNECTOR_BUNDLED_CONFIGS/bundles.json $JUPYTER_CONFIG_DIR/nbconfig/sparkconnector_bundles.json
     ln -s $CONNECTOR_BUNDLED_CONFIGS/spark_options.json $JUPYTER_CONFIG_DIR/nbconfig/sparkconnector_spark_options.json
```

Comment:
It is clear that the bundles (sparkconnector_bundles.json) should be provided just for LCG, not custom envs.
However, the Spark options used for autocomplete (sparkconnector_spark_options.json) can be useful also for the customenvs case, if the user wants to add some extra spark option.

This means we need to rethink this. Where is `CONNECTOR_BUNDLED_CONFIGS` defined?

Reply:
Apparently nowhere in the Swan code I have access to. The only scripts in which it is used are:
- [swan-cern/scripts/before-notebook.d/10_spark.sh](https://github.com/swan-cern/jupyter-images/blob/6bd8470af1a4382db779faedd02b6d6db8f5561b/swan-cern/scripts/before-notebook.d/10_spark.sh#L39)
- [userconfig.sh](https://github.com/swan-cern/systemuser-image/blob/4c408a7a278026ef79dbbd3da941316db6271ac1/userconfig.sh#L69)

Reply:
I searched for it in the spark source code and I didnt find anything as well

Reply:
It's in hadoop-confext, see my reply on mattermost.